src/js/models/home.js

```diff
@@ -98,8 +101,28 @@ export default class Home{
 		}
 	}
 
+	async getAndRenderJunior(juniorUid, renderSeniorOppDetail,junior,info){
+		try{			
+			let juniorInfo,juniorPic;
+			//console.log(juniorUid);
+			await db.collection('Profiles').doc(juniorUid).get().then(doc => {
+				juniorInfo = doc;
+			});
+			await db.collection('Images').where("userId","==",juniorUid).get().then(docs => {
+				docs.forEach(doc => {
+					juniorPic = doc.data().url;
+					console.log()
+				})
+			});
+			renderSeniorOppDetail(junior.data().status, junior.data().uid, info, juniorInfo, juniorPic);

```

Comment:
Can we do:
```suggestion
			renderSeniorOppDetail(junior.data().status, juniorUid, info, juniorInfo, juniorPic);

```

Reply:
If so, it's better to just pass `junior.data().status` instead of the whole `junior` object to this function since `junior` is only used for its status in this function

Reply:
Done